edumap/app/models/lesson.rb

```diff
@@ -35,7 +35,15 @@ class Lesson < ActiveRecord::Base
   scope :with_created_at_gte, -> (ref_date) { where('created_at >= ?', ref_date) }
 
   scope :with_standard, -> (standards) { with_association(:standards, standards) }
-  scope :with_grade, -> (levels) { with_association(:levels, levels) }
+  scope :with_grade, -> (level) do
+    grade_at_most(level).grade_at_least(level)
```

Comment:
A super tiny nitpick: I think this is small enough that we can get away with not splitting it into submethods, and I think that would be a little clearer and cleaner.